src/employees/addEmployee.js

```diff
@@ -1,160 +1,141 @@
-import React, { Component } from 'react';
-
+import React, { Component } from "react";
+
+const defaultState = {
+  id: "",
+  name:"",
+  company:"",
+  salary:"",
+  age:"",
+  phone:"",
+  email:""
+}
 class AddEmployee extends Component {
  constructor (props){
    super(props);
    this.state = {
-     id: '',
-     name:'',
-     company:'',
-     salary:'',
-     age:'',
-     phone:'',
-     email:''
+     ...defaultState
    };
 
-   this.updateId = this.updateId.bind(this);
-   this.updateName = this.updateName.bind(this);
-   this.updateCompany = this.updateCompany.bind(this);
-   this.updateSalary = this.updateSalary.bind(this);
-   this.updateAge = this.updateAge.bind(this);
-   this.updatePhone = this.updatePhone.bind(this);
-   this.updateEmail = this.updateEmail.bind(this);
+   this.handleChange = this.handleChange.bind(this);
    this.add = this.add.bind(this);
 
  }
 
- updateId(e){
+ handleChange (e) {
    this.setState({
-     id: e.target.value
+     [e.target.name]: e.target.value
```

Comment:
Con la propiedad name del target, nos ahorramos todos los metodos de `update`, es muy útil en este tipo de formularios.

Puedes leer un poco sobre eso acá:

https://reactjs.org/docs/forms.html#handling-multiple-inputs

Adicional, es buena práctica llamar a tus handlers `handleAlgo` en este caso `handleChange` y a tus eventos `onAlgo` por lo que va a ser común encontrarte componentes como `<Componente onEvento={this.handleEvento} />`